cloudantv1/cloudant_v1_integration_test.go

```diff
@@ -0,0 +1,118 @@
+// © Copyright IBM Corporation 2020. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudantv1_test
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/IBM/go-sdk-core/v4/core"
+	"github.ibm.com/cloudant/cloudant-go-sdk/cloudantv1"
```

Comment:
```suggestion
	"github.com/IBM/cloudant-go-sdk/cloudantv1"
```

Reply:
Thank you for the suggestion. Committed.